ocp4/profiles/coreos-ncp.profile

```diff
@@ -189,16 +189,13 @@ selections:
     ### Install Required Packages
     - package_sssd-ipa_installed
     - package_aide_installed
-    #- package_dnf-automatic_installed
     - package_firewalld_installed
     - package_iptables_installed
     - package_libcap-ng-utils_installed
-    #- package_openscap-scanner_installed
```

Comment:
Needed to config scan the system

Reply:
@shawndwells this is a brand new rule entirely as you cannot use yum/dnf.

Reply:
Added the rule for discussion later